lib/jekyll-import/importers/mt.rb

```diff
@@ -243,6 +228,34 @@ def self.suffix(entry_type)
           entry_type
         end
       end
+
+      def self.database_from_opts(opts)
+        engine   = options.fetch('engine', 'mysql')
+        dbname   = options.fetch('dbname')
+        user     = options.fetch('user')
+        pass     = options.fetch('password', "")
+        host     = options.fetch('host', "localhost")
+
+        db_connect_opts = {
```

Comment:
Probably not much point building this outside of the MySQL/PostgreSQL case. Or fetching the user/password/host options, really.